src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,17 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value <= 0) {
+            return "0";
```

Comment:
that's not included in tests, but does that make sense return  "0" if input will be -20 ? 
if suggest we change if condition a bit

Reply:
Done